dinero/gateways/authorizenet_gateway.py

```diff
@@ -553,7 +553,9 @@ def create_customer(self, options):
             if error_code == 'E00039':  # Duplicate Record
                 e.customer_id = None
 
-                customer_match = re.search(r'^A duplicate record with ID (.*) already exists.$', e.message[0][1])
+                customer_match = re.search(
+                    r'A duplicate record with ID (.*) already exists.', str(e)
+                )
```

Comment:
I don't understand, but I could me mistaking. `e.message` is suppose to be a list of all the collected error messages. It looks like you're trying to find a string in the repr of the dict? That's not very optimal. How is that related to python 3?